components/StyledButton.js

```diff
@@ -0,0 +1,18 @@
+import styled, { css } from "styled-components";
```

Comment:
```suggestion
import styled from "styled-components";
```

you aren't using { css } here.